src/ceph-disk

```diff
@@ -1977,6 +2014,8 @@ def activate(
                 )
             if conf_val is not None:
                 init = conf_val
+            elif is_systemd():
+                init = 'systemd'
```

Comment:
This specific portion can be avoided if specified in the configuration file. I don't see where we would fail if the machine was `systemd` 

Making that `if` condition just for Ubuntu seemed like it was not going to be the right approach since it is not the only distribution out there that uses (or will use) systemd.

If something breaks because there is a distribution that is `systemd` and `ceph-disk` is doing the wrong thing we should fix that then